storage/sql/migrate.go

```diff
@@ -190,4 +190,13 @@ var migrations = []migration{
 			);`,
 		},
 	},
+	{
+		stmts: []string{`
+			alter table client
+				add column saml_init_redirect_uri text not null default '';`,
+			`
+			alter table client
+				add column saml_init_scopes bytea; -- JSON array of strings`,
```

Comment:
@srenatus i took a stab at updating the storage connectors with my changes. tests are passing now, but one questionable bit is this migration. seems like the convention is to make all columns `not null` across every table, but i ran into a bit of a conflict here.

postgresql won't let you add a `not null` column to a table without specifying a default, however mysql won't let you specify a default on a `BLOB` column (what `bytea` translates to in the mysql flavor). so.... i just let this column be nullable. not sure if there's any potential issues with that.

Reply:
We'd expect most clients _not_ to have a SAML config, wouldn't we? So, null seems alright for that...? 🤔